docs/source/contributing/using_gitpod.md

```diff
@@ -0,0 +1,126 @@
+
+## Using Gitpod to Contribute to PyMC
+
+These instructions are for contributing specifically to the [pymc-devs/pymc](https://github.com/pymc-devs/pymc) repo.
+
+### Gitpod Workflow
+
+1. Fork the pymc repo: [https://github.com/pymc-devs/pymc](https://github.com/pymc-devs/pymc)
+
+2. Create a Gitpod account. You can login and authorize access via your GitHub account:  [https://gitpod.io/](https://gitpod.io/)
+
+:::{note}
+Gitpod will show up as an authorized application in your GitHub account here: [https://github.com/settings/applications](https://github.com/settings/applications)
+:::
+
+3. Grant GitHub / Gitpod integration permissions.
+
+    a) Go to: [https://gitpod.io/user/integrations](https://gitpod.io/user/integrations)
+
+    b) Select GitHub and then "edit permissions"
+
+    c) Select these permission: user:email, public_repo, repo, workflow
+
+:::{figure-md} Gitpod integration
+
+![gitpod_integration](gitpod/gitpod_integration.png)
+
+Gitpod [integration options](https://gitpod.io/user/integrations)
+:::
+
+4. Within Gitpod, create a ["New Workspace"](https://gitpod.io/workspaces).  Here you will want to select the forked pymc repo. If you don't see it, you can paste into the "Context URL" your forked repo path.  For example:  `https://github.com/yourusername/pymc`.  Then select "New Workspace".
+
+:::{figure-md} Gitpod workspace
+
+![gitpod_workspace](gitpod/gitpod_workspace.png)
+
+Gitpod [workspace](https://gitpod.io/workspaces)
+:::
+
+:::{note}
+Gitpod will pull a container and set up the workspace.  It will take a few minutes for the container to build.
+:::
+
+5. Once Gitpod is up and running, the interface is similar to a Visual Studio Code (VSC) interface, which will appear in your browser. You will observe installation notices in the terminal window.  This can take 5-10 minutes. Once that is complete, the terminal will indicate you are on the "(base)" environment on Gitpod with your forked repo.
+
+    Here is an example:
+
+    ```bash
+    (base) gitpod@reshamas-pymc-0ygu5rf74md:/workspace/pymc$
+    ```
+
+:::{note}
+This working environment has been set up with [micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) which is a small, pure-C++ executable with enough functionalities to bootstrap fully functional conda-environments.
+:::
+
+6. Check that your git remotes are correct with `git remote -v` at the terminal.
+
+    Example:
+
+    ```bash
+    (base) gitpod@reshamas-pymc-0ygu5rf74md:/workspace/pymc$ git remote -v
+    origin  https://github.com/reshamas/pymc.git (fetch)
+    origin  https://github.com/reshamas/pymc.git (push)
+    upstream        https://github.com/pymc-devs/pymc.git (fetch)
+    upstream        https://github.com/pymc-devs/pymc.git (push)
+    (base) gitpod@reshamas-pymc-0ygu5rf74md:/workspace/pymc$
+    ```
+
+7. Check which version of python and pymc are being used at the terminal.
+
+    * Check version of pymc: `pip list | grep pymc`
+
+    Example:
```

Comment:
```suggestion
    a) Check version of pymc: `pip list | grep pymc`. Example:
```

This might be a bit more consistent with the sub items above. Otherwise, the "Example" and code lines should probably be indented at the same level as the text in "Check version...", not at the same level as the "*"